flexible-classroom/develop/record-a-class.md

```diff
@@ -25,8 +25,7 @@ If you want to implement recording on your own, you can refer to the following d
 
 ![implement recording on your own](https://web-cdn.agora.io/docs-files/1638259107675)
 
-When you deploy the web page to be recorded into your CDN, you can use the template HTML file `templates/record_page_prod.html` provided by Agora in the [Flexible-Classroom-Desktop](https://github.com/AgoraIO-Community/flexible-classroom-Desktop) GitHub repository.
-
+When you deploy the web page to be recorded into your CDN, you can use the template HTML file `templates/record_page_prod.html` provided by Agora in the [Flexible-Classroom-Desktop](https://github.com/AgoraIO-Community/flexible-classroom-desktop) GitHub repository.
```

Comment:
This doc is the same for all platforms, which means flexible-classroom-desktop will show also for Android and iOS, which is wrong. Probably a good place for conditional text